stable/rabbitmq-ha/templates/statefulset.yaml

```diff
@@ -81,6 +86,8 @@ spec:
               touch "${RABBITMQ_MNESIA_DIR}/force_load"
             fi
             {{- end }}
+            chown {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.runAsGroup }} /var/lib/rabbitmq/.erlang.cookie
```

Comment:
This should check if the user and group are non-null since securityContext can be set to null/empty

Reply:
Correct. Updated